catalog/transformer/catalog-transformer-rtf/src/main/java/ddf/catalog/transformer/output/rtf/model/ExportCategory.java

```diff
@@ -97,6 +99,10 @@ public void setAttributes(List<String> attributes) {
   }
 
   private String attributeKeyFrom(String key) {
+    if (key.startsWith(EXTENDED_ATTRIBUTE_PREFIX)) {
+      key = key.replace(EXTENDED_ATTRIBUTE_PREFIX, "");
```

Comment:
❗️  We should probably only strip the first `EXTENDED_ATTRIBUTE_PREFIX.length()` characters so we don't unwittingly modify attributes. For example `next.attribute` -> `nattribute`

Reply:
I don't think we conventionally support that particular attribute name
https://github.com/codice/ddf/blob/767cb947935a83fc1f9b1fdde38bc09f605ecfd0/distribution/docs/src/main/resources/content/_architectures/_data/data-intro.adoc

Reply:
From https://github.com/codice/ddf/blob/767cb947935a83fc1f9b1fdde38bc09f605ecfd0/distribution/docs/src/main/resources/content/_architectures/_data/data-intro.adoc#attribute-naming-conventions
>Catalog taxonomy elements follow the naming convention of `group-or-namespace.specific-term`, except for extension fields outside of the core taxonomy. These follow the naming convention of `ext.group-or-namespace.specific-term` and must be namespaced. Nesting is not permitted.

This can happen if the `group-or-namespace` ends with `ext`